codalab/rest/interpret.py

```diff
@@ -68,7 +62,7 @@ def _interpret_search():
     }
     ```
     """
-    return interpret_search(request.json)
```

Comment:
Why is this being modified?

Reply:
I can't find any method with name interpret_search, and both PyCharm and Flake8 are complaining.